pymultitor.py

```diff
@@ -280,7 +280,7 @@ def load(loader):
             name="tor_timeout",
             typespec=int,
             default=DEFAULT_INIT_TIMEOUT,
-            help="number of seconds before our attempt to start a tor instance timed out",
+            help="number of seconds before our attempt to start a tor instance timed out, no timeouts are applied if 0",
```

Comment:
Please change the help message to "timeout in seconds for starting a tor instance; 0 disables timeout"